scripts/install.sh

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env sh
+_sudo () {
```

Comment:
This isn't being invoked anywhere yet

Reply:
It is here already https://github.com/kubeshop/kgw/blob/main/scripts/install.sh#L63